packages/ppx/src/declarations_to_emotion.re

```diff
@@ -561,20 +587,25 @@ let render_named_color =
 let render_color_alpha =
   fun
   | `Number(number) => [%expr `num([%e render_number(number)])]
-  | `Percentage(percentage) => [%expr
-      `percent([%e render_number(percentage /. 100.0)])
-    ];
+  | `Extended_percentage(`Percentage(pct)) => render_percentage(pct /. 100.0)
+  | `Extended_percentage(pct) => render_extended_percentage(pct);
 
 let render_function_rgb = ast => {
-  let to_number = percentage => percentage *. 2.55;
+  let color_to_float = v => render_integer(v |> int_of_float);
+
+  let to_number = fun
+    // TODO: bs-css rgb(float, float, float)
+    | `Percentage(pct) => color_to_float(pct *. 2.55)
+    | _ => render_integer(999);
```

Comment:
Is this unreachable? If so, isnt better to fail here? So it would be easy to catch some possible error in the future

Reply:
Good catch, was missing a function_calc here!